CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# 1.31
```

Comment:
```suggestion
1.31
====
```

like below?

Reply:
working on a automated tool to handle our changelogs for us across the `symfony/*-bundle` repos and a few repos outside of `symfony/*` e.g. `symfonycasts/*`. the `#` && `===` deliver the same result from a ux perspective.. havent even thought about which of the 2 is "best" when compared across all of the repos.